game/Submods/NSFW Submod/nsfw_stories.rpy

```diff
@@ -445,81 +448,14 @@ label nsfw_monika_erotic_stories_init:
                     
     $ pushEvent("nsfw_erotic_story_natsuki_deepthroat1", skipeval=True)
     return
-       
-label nsfw_monika_erotic_stories_2:
-    m 1eua "Sure, [player]!"
-    m 1eua "Which story would you like me to tell?"
-    
-    $ _history_list.pop()
-    if renpy.seen_label("nsfw_erotic_story_natsuki_deepthroat"):
-        menu:
-            "Natsuki Deepthroat":
-                m 1eua "Alright!"
-                m 1eua "So, to provide context again, Natsuki was being a pain at the time."
-                m 1eua "So I thought I'd treat her to some..."
-                m 1eua "{i}Rough throating action{/i}."
-                m 1eua "Ahaha~"
-                call nsfw_erotic_story_natsuki_deepthroat
-                return
-
-            "Any other erotic stories?":
-                
-                call nsfw_erotic_story_sayori_ballscleaning
-                return
-
-    elif renpy.seen_label("nsfw_erotic_story_sayori_ballscleaning"):
-        menu:
-            "Natsuki Deepthroat":
-                m 1eua "Alright!"
-                m 1eua "So, to provide context again, Natsuki was being a pain at the time."
-                m 1eua "So I thought I'd treat her to some..."
-                m 1eua "{i}Rough throating action{/i}."
-                m 1eua "Ahaha~"
-                call nsfw_erotic_story_natsuki_deepthroat
-                return
-
-            "Sayori Ballcleaning":
-                # add prelude here
-                call nsfw_erotic_story_sayori_ballscleaning
-                return
-
-            "Any other erotic stories?":
-                
-                call nsfw_erotic_story_yuri_titjob1
-                return
-    
-    elif renpy.seen_label("nsfw_erotic_story_yuri_titjob1"):
-        menu:
-            "Natsuki Deepthroat":
-                m 1eua "Alright!"
-                m 1eua "So, to provide context again, Natsuki was being a pain at the time."
-                m 1eua "So I thought I'd treat her to some..."
-                m 1eua "{i}Rough throating action{/i}."
-                m 1eua "Ahaha~"
-                call nsfw_erotic_story_natsuki_deepthroat
-                return
-
-            "Sayori Ballcleaning":
-                # add prelude here
-                call nsfw_erotic_story_sayori_ballscleaning
-                return
-
-            "Yuri Titjob":
-                # add prelude here
-                call nsfw_erotic_story_yuri_titjob1
-                return
-
-            "Any other erotic stories?":
-                # add prelude here
-                return  
 
 # Thanks for the erotic story, KittyTheCocksucker
 init 6 python:
     addEvent(
         Event(
             persistent._nsfw_story_database,
             eventlabel="nsfw_erotic_story_natsuki_deepthroat1",
-            prompt="Natsuki deepthroat",
+            prompt="Natsuki Deepthroat",
```

Comment:
We want to keep the second word uncapitalized to keep it consistent with the rest of MAS